### DISCUSSION TITLE
data compatibility issues in rollback scenarios

### DISCUSSION THREAD

**Discussion by chenshi5012** [minio/minio#17152] [Q&A] [answered]
hello, guys

@harshavardhana 

please check this mr: 

https://github.com/minio/minio/pull/11758

since xl.meta and part.1 file  merge into one file.  when minio rollback ， the file wont be able to be read.   Is there any way to separate them.

**Comment by harshavardhana** [comment, marked as answer]
@chenshi5012 rollbacks are never supported in MinIO, you always move forward.

**Reply by chenshi5012** [reply]
thanks. we do not have enough space  to backup metadata.
